TS03-Subset-Data-and-No-Data-Values.Rmd

```diff
@@ -165,7 +162,7 @@ qplot (datetime, prec,
 ## Missing Data - Dealing with data gaps
 
 ###Checking for No Data Values
-In the <a href="XXX" target="_blank"> Understand The Data</a>
+In the  [{{site.baseurl}} / LINKHERE](Lesson 01 - )
```

Comment:
that one is for @lwasser!
i just copied what I saw in other parts of the scripts